doc/classes/ResourceImporterDynamicFont.xml

```diff
@@ -1,43 +1,77 @@
 	<brief_description>
+		Imports a TTF, OTF, WOFF or WOFF2 font file for font rendering that adapts to any size.
 	</brief_description>
 	<description>
+		Unlike bitmap fonts, dynamic fonts can be resized to any size and still look crisp. Dynamic fonts also optionally support MSDF font rendering, which allows for run-time scale changes with no re-rasterization cost.
+		While WOFF and especially WOFF2 tend to result in smaller file sizes, there is no universally "better" font format. In most situations, it's recommended to use the font format that was shipped on the font developer's website.
+		See also [ResourceImporterBMFont] and [ResourceImporterImageFont].
 	</description>
 	<tutorials>
+		<link title="Dynamic fonts - Using fonts">$DOCS_URL/tutorials/ui/gui_using_fonts.html#dynamic-fonts</link>
 	</tutorials>
 	<members>
 		<member name="allow_system_fallback" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], automatically use system fonts as a fallback if a glyph isn't found in this dynamic font. This makes supporting CJK characters or emoji more straightforward, as you don't need to include a CJK/emoji font in your project. See also [member fallbacks].
+			[b]Note:[/b] The appearance of system fonts varies across platforms. Loading system fonts is only supported on Windows, macOS, Linux, Android and iOS.
 		</member>
 		<member name="antialiasing" type="int" setter="" getter="" default="1">
+			The font antialiasing method to use.
+			[b]Disabled:[/b] Most suited for pixel art fonts, although you do not [i]have[/i] to change the antialiasing from the default [b]Grayscale[/b] if the font file was well-created and the font is used at an integer multiple of its intended size. If pixel art fonts have a bad appearance at their intended size, try setting [member subpixel_positioning] to [b]Disabled[/b] instead.
+			[b]Grayscale:[/b] Use grayscale antialiasing. This is the approach used by the operating system on macOS, Android and iOS.
+			[b]LCD Subpixel:[/b] Use antialiasing with subpixel patterns to make fonts sharper on LCD displays. This is the approach used by the operating system on Windows and most Linux distributions. The downside is that this can introduce "fringing" on edges, especially on display technologies that don't use standard RGB subpixels (such as OLED displays). The LCD subpixel layout is globally controlled by [member ProjectSettings.gui/theme/lcd_subpixel_layout], which also allows falling back to grayscale antialiasing.
 		</member>
 		<member name="compress" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], uses lossless compression for the resulting font.
 		</member>
 		<member name="fallbacks" type="Array" setter="" getter="" default="[]">
+			List of font fallbacks to use if a glyph isn't found in this dynamic font. Fonts are the beginning of the array are attempted first. See also [member allow_system_fallback].
```

Comment:
> Fonts are the beginning of the array are attempted first.

This is not true, at least for `TextServerAdvanced` which will prefer fonts with `script_support` and `language_support` matching missing glyph run first (including both font metadata and overrides specified by user).